src/promptflow/promptflow/_cli/_pf/_flow.py

```diff
@@ -37,9 +37,9 @@
     copy_extra_files,
 )
 from promptflow._cli._utils import _copy_to_flow, activate_action, confirm, inject_sys_path, list_of_dict_to_dict
-from promptflow._constants import FlowLanguage
+from promptflow._constants import ConnectionProviderConfig, FlowLanguage
 from promptflow._sdk._configuration import Configuration
-from promptflow._sdk._constants import PROMPT_FLOW_DIR_NAME, ConnectionProvider
+from promptflow._sdk._constants import PROMPT_FLOW_DIR_NAME
```

Comment:
from promptflow._constants import PROMPT_FLOW_DIR_NAME ?

there is another PROMPT_FLOW_DIR_NAME there

Reply:
I'll let sdk constants reference the promptflow constants one